Dockerfile

```diff
@@ -17,7 +17,6 @@ RUN curl -sL https://deb.nodesource.com/setup_10.x | bash - \
 
 RUN npm install -g npm@6.3.0
 
-RUN gem install bundler \
```

Comment:
Why remove bundler installation @oriolgual @mrcasals ?
Is it already installed by default with ruby 2.7.1 image?

Reply:
Looks like it's now [part of the standard library](https://ruby-doc.org/stdlib-2.7.0/libdoc/bundler/rdoc/Bundler.html) (since 2.6).

The [Dockerfile smoke tests it here](https://github.com/docker-library/ruby/blob/5a4e2d8d9fae6681ac3e2344eac547f0e2cdc598/2.7/buster/Dockerfile#L78).

Reply:
Looking at the standard library web page I could not get which `bundler` version is being installed. We need it to be higher than v2.

Reply:
With the ruby:2.6.6 image bundler is indeed at 1.7.2, nice catch @tramuntanal:

```bash
➜  decidim-docker git:(sandbox) docker run --name ruby -i -t ruby:2.6.6 bash
root@2ce5d8d66e46:/# ruby -v
ruby 2.6.6p146 (2020-03-31 revision 67876) [x86_64-linux]
root@2ce5d8d66e46:/# bundle -v
Bundler version 1.17.2
```

Reply:
Good!